README.md

```diff
@@ -1,23 +1,43 @@
+# Gradle static analysis plugin
+[![](https://ci.novoda.com/buildStatus/icon?job=gradle-static-analysis-plugin)](https://ci.novoda.com/job/gradle-static-analysis-plugin/lastSuccessfulBuild) [![](https://img.shields.io/badge/License-Apache%202.0-lightgrey.svg)](LICENSE.txt) [![Bintray](https://api.bintray.com/packages/novoda/maven/gradle-static-analysis-plugin/images/download.svg)](https://bintray.com/novoda/maven/gradle-static-analysis-plugin/_latestVersion)
 
-A Gradle plugin to easily apply the same setup of static analysis tools across different Android or Java projects.<br/>
+A Gradle plugin to easily apply the same setup of static analysis tools across different Android or Java projects.
 
 ## Description
-
-Gradle supports many popular static analysis (Checkstyle, PMD, FindBugs, etc) via a set of built-in
-plugins. Using these plugins in an Android module will require an additional setup to compensate for the differences between
-the model adopted by the Android plugin compared to the the Java one.<br/>
+Gradle supports many popular static analysis (Checkstyle, PMD, FindBugs, etc) via a set of built-in plugins.
+Using these plugins in an Android module will require an additional setup to compensate for the differences between
+the model adopted by the Android plugin compared to the the Java one.
 
 The `gradle-static-analysis-plugin` aims to provide:
-- flexible, configurable penalty strategy for builds,
-- easy, Android-friendly integration for all static analysis,
-- convenient way of sharing same setup across different projects,
-- healthy, versionable and configurable defaults.
+- flexible, configurable penalty strategy for builds
+- easy, Android-friendly integration for all static analysis
+- convenient way of sharing same setup across different projects
+- healthy, versionable and configurable defaults
+
+### Supported static analysis tools
+The plugin supports various static analysis tools for Java, Kotlin and Android projects:
+
+ * [`Checkstyle`](https://checkstyle.sourceforge.net)
+ * [`PMD`](https://pmd.github.io)
+ * [`FindBugs`](http://findbugs.sourceforge.net/)
+ * [`Detekt`](https://github.com/arturbosch/detekt)
+
+Support for additional tools is planned but not available yet:
 
-## Adding to your project
+ * [`KtLint`](https://github.com/shyiko/ktlint)
+ * [`Android Lint`](https://developer.android.com/studio/write/lint.html)
+
+Please note that the tools availability depends on the project the plugin is applied to. For more details please refer to the
+[supported tools](docs/supported-tools.md) page.
+
+### Out-of-the-box support for Android and Kotlin projects
```

Comment:
Title includes `kotlin` but the content does not mention anything kotlin related.

Reply:
Not much to mention besides that? I was not sure what to add. Suggestions welcome.

Reply:
If we don't, maybe we can remove `Kotlin` from this header? Having support for `detekt` `ktlint` already indicates that we support those tools.

Reply:
Well, does it imply it just as much as mentioning Android Lint implies support for Android projects? I don't _really_ see the problem here, to be honest.

Reply:
I agree with @tasomaniac, imho we could remove `Kotlin` from the headline. Additionally we could be a bit more explicit by mentioning `PMD`, `Findbugs` and `Checkstyle` as problematic when it comes to Android Projects.

Reply:
We have a special paragraph to talk about `Android` because the default tools needs setup for Android and does not work out of the box. 
That's why this paragraph is really good. 
We don't have anything special around Kotlin, imo

Reply:
cb7a7d4eae55d8d31d7642abf765a49a2c755257